client/src/api/BizMarketApi.js

```diff
@@ -1,7 +1,5 @@
-import React from "react";
+import axios from "axios";
 
-const BizMarketApi = () => {
-	return <div></div>;
-};
-
-export default BizMarketApi;
+export default axios.create({
+	baseURL: "http://localhost:3100/api",
```

Comment:
@zhkvivan Can we change this line. The reason is that this will work when running locally but will fail once we push to heroku. I think the best thing to do would be to be able to tell between dev/prod config. If dev then baseURL=<localhost>, else baseURL=<heroku-url>. @PeterArmstrong79 We will also need to do similar in the backend for this when connecting to the DB

Reply:
@LiamGve ok good point. But how can we get permanent heroku link? that link that Juby gave us isn't working now

Reply:
@zhkvivan good point, but even for now if we have the code to distinguish between local and not local then once we have a permanent link we can just paste it in

Reply:
I'm not 100% sure how it's done with React (perhaps one of the other TAs with more exp in React might know) but I can see files `client/webpack/dev.config.js` and `client/webpack/prod.config.js` that might be able to help us out here. In the package.json we will load 1 of these files depending on if it's `npm run dev:client` for local dev or if it's `npm run build:client` which is what I assume is used to deploy to heroku

Reply:
There is an environment variable in node called NODE_ENV === 'production'. This is normally done on the server side. Or you can set an environment variable in react for the backend side. To set an envinment variable in react create a file called .env in that file call the variable REACT_APP_URI, then when calling into a file call process.env.REACT_APP_URI

Reply:
@LiamGve done now